bindata/etcd/cluster-restore.sh

```diff
@@ -67,6 +67,15 @@ function wait_for_containers_to_stop() {
   done
 }
 
+function check_snapshot_status() {
+  SNAPFILE="$1"
```

Comment:
we have standardized on`local varname="$1"`

https://github.com/openshift/cluster-etcd-operator/blob/c2cab02a3861e5a68d6f4fdc82903273c906d4da/bindata/etcd/cluster-backup.sh#L87

Reply:
Sorry I was not more clear but that standard we have been trying to adopt is the use of lowercase local variables inside of functions.

Ref: https://tldp.org/LDP/abs/html/localvar.html

Reply:
I had thought about it too, but since other functions already had uppercase local variables, I went with the current flow.

I can change all of them to lowercase too.